skimage/util/process.py

```diff
@@ -0,0 +1,68 @@
+from multiprocessing import cpu_count
+
+import dask.array as da
+
+__all__ = ['process_chunks']
+
+
+def _get_chunks(shape, ncpu):
```

Comment:
@jni About your second point: this would make the last remainder smaller. I want to avoid this because you could accidentally have your boundary depth be larger than the last chunk. If we just extend the last chunk by the remainder, we know the safe upper bound for boundary depth would be `min(shape) // ciel(ncpu ** (1./len(shape))) `